Dockerfile

```diff
@@ -1,41 +1,86 @@
-FROM odoo:11.0
+FROM debian:stretch-20200130-slim
 MAINTAINER numigi <contact@numigi.com>
 
-USER root
+# Generate locale C.UTF-8 for postgres and general locale data
+ENV LANG C.UTF-8
+
+# Set the version of Odoo
+ENV ODOO_VERSION 11.0
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        curl \
+        dirmngr \
+        fonts-noto-cjk \
         gcc \
         git-core \
+        gnupg \
+        libldap2-dev \
+        libsasl2-dev \
+        libssl-dev \
+        node-less \
         python3-dev \
-    && \
-    git config --global user.name "Odoo" && \
-    git config --global user.email "root@localhost" && \
-    rm -rf /var/lib/apt/lists/*
+        python3-pip \
+        xz-utils \
+    && curl -o wkhtmltox.deb -sSL https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.stretch_amd64.deb \
+    && echo '7e35a63f9db14f93ec7feeb0fce76b30c08f2057 wkhtmltox.deb' | sha1sum -c - \
+    && apt-get install -y --no-install-recommends ./wkhtmltox.deb \
+    && rm -rf /var/lib/apt/lists/* wkhtmltox.deb
 
-RUN pip3 install pip==9.0.1 wheel==0.30.0
+# Install latest postgresql-client (copied from the official odoo image)
+RUN set -x; \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/numigi/odoo-public/pullRequest?prid=5035243)